event-service/src/main/resources/application.properties

```
@@ -0,0 +1,12 @@
+server.port=9001
```

9001 trùng với user-service r
```suggestion
server.port=9004
```